packages/cli/core/package.json

```diff
@@ -75,14 +75,14 @@
   },
   "peerDependencies": {
     "@angular/cli": "^15.0.0",
-    "@vue/cli": "^5.0.8",
```

Comment:
Is the vue cli not needed as a peer dep anymore?

Reply:
Indeed, we stopped using it since the CLI v2, it's mostly just some cleaning